src/test/kotlin/uk/gov/dluhc/printapi/rest/DeprecatedGetTemporaryCertificateSummariesByApplicationIdIntegrationTest.kt

```diff
@@ -0,0 +1,123 @@
+package uk.gov.dluhc.printapi.rest
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.http.MediaType
+import uk.gov.dluhc.printapi.config.IntegrationTest
+import uk.gov.dluhc.printapi.database.entity.SourceType
+import uk.gov.dluhc.printapi.models.TemporaryCertificateStatus
+import uk.gov.dluhc.printapi.models.TemporaryCertificateSummariesResponse
+import uk.gov.dluhc.printapi.testsupport.bearerToken
+import uk.gov.dluhc.printapi.testsupport.testdata.anotherValidEroId
+import uk.gov.dluhc.printapi.testsupport.testdata.entity.buildTemporaryCertificate
+import uk.gov.dluhc.printapi.testsupport.testdata.getBearerToken
+import uk.gov.dluhc.printapi.testsupport.testdata.model.buildElectoralRegistrationOfficeResponse
+import uk.gov.dluhc.printapi.testsupport.testdata.model.buildTemporaryCertificateSummary
+import java.time.ZoneOffset.UTC
+import java.time.temporal.ChronoUnit.SECONDS
+
+@Deprecated(
+    """
+        The endpoint /eros/{ERO_ID}/temporary-certificates/applications/{APPLICATION_ID} has been deprecated 
+        and this test class should be removed when the implementation is removed.    
+    """
+)
+internal class DeprecatedGetTemporaryCertificateSummariesByApplicationIdIntegrationTest : IntegrationTest() {
```

Comment:
This class is actually just a rename of `GetTemporaryCertificateSummariesByApplicationIdIntegrationTest` with the `@Deprecated` annotation added